src/main/java/hudson/plugins/git/extensions/impl/AuthorInChangelog.java

```diff
@@ -17,13 +17,49 @@
     public AuthorInChangelog() {
     }
 
+    /**
+     * {@inheritDoc}
+     */
     @Override
     public boolean requiresWorkspaceForPolling() {
         return true;
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        return o instanceof AuthorInChangelog;
```

Comment:
Doesn't the check:

> getClass() != o.getClass()

guarantee that o is an instanceof AuthorInChangelog?  If so, isn't this more readily expressed as "return true"?  Or is this worrying about the risk of a class which might inherit from this class and fail to override the equals method?

I think it is correct that every instance of AuthorInChangelog is equal to every other instance of AuthorInChangelog, since the AuthorInChangelog class is being used as a boolean.  If AuthorInChangelog is enabled, then the author should be used rather than the committer when displaying changelogs.